tests/echo/host/host.c

```diff
@@ -17,7 +17,7 @@ int host_echo(char* in, char* out, char* str1, char* str2, char str3[100])
     OE_TEST(strcmp(str2, "oe_host_strdup2") == 0);
     OE_TEST(strcmp(str3, "oe_host_strdup3") == 0);
 
-    strcpy(out, in);
+    strcpy_s(out, 100, in);
```

Comment:
Hard coding 100 here is dangerous, since if someone later changes the caller's length there might be memory corruption.  Safer to have the caller pass in the length.

Reply:
The 100 is hard-coded in the EDL file. I will change it to sizeof(out) first.